api_get.py

```diff
@@ -0,0 +1,66 @@
+from sanic import Sanic
+from sanic import response
+from serializers import *
+from functions import *
+from sanic.response import text
+import requests
+
+app = Sanic('some_name')
+
+
+# GET
+
+@app.route('/')
+async def get_handler(request):
+    return response.json({'message': 'Hello world!'}, headers={'Service': 'SDA'}, status=200)
+
+
+@app.route('/payment')
+async def get_all(request):
+    payment_rows = await get_all_payments()
```

Comment:
What do you mean under "row" word? I think we should find better one. What about:
```
payments = await get_all_payments()
```

Reply:
changed.